app/controllers/admin/users_controller.rb

```diff
@@ -14,10 +14,10 @@ def new
 
   def create
     @user = User.new(user_params)
-    create_user
-
-    return redirect_to admin_user_path(@user) if @user.persisted?
-
+    if @user.valid?
```

Comment:
Do we have a feature spec for invalid users?

Reply:
I beleive so. ```Adding a user with Rails validation error``` 

spec/features/adding_a_user_spec.rb:24

I am trying to stop the call to Auth0 unless the form passes the Rails validation first, that is what I was seeing until I added this code.